docs/AnaplanExport-batchsource.md

```diff
@@ -0,0 +1,139 @@
+
+If the plugin is not run on a Dataproc cluster, the path to a service account key must be provided.
+The service account key can be found on the Dashboard in the Cloud Platform Console.
+Make sure the account key has permission to access BigQuery and Google Cloud Storage.
+The service account key file needs to be available on every node in your cluster and
+must be readable by all users running the job.
+
+Anaplan Properties
+----------
+**Workspace Id:** The target Anaplan workspace ID.
+ 
+**Model Id:** The target Anaplan model ID.
+
+**Server File Name:** The name of the predefined placeholder file name in the model for this data import.
+
+**Service Location:** The root service location of the Anaplan API.
+Default: https://google.anaplan.com
+ 
+**Auth Service Location:** The service location for the authentication.
+Default: https://google.anaplan.com
+
+**User Name:** The service account used for the connection.
+
+**Password:** The password for authentication. It's suggested to use CDF build-in Cloud KMS integration 
+to manage the password and utilize macro in this filed. 
+Reference: [Provide secure information to pipelines](https://datafusion.atlassian.net/wiki/spaces/KB/pages/32276556/Provide+secure+information+to+pipelines)
+
+GCS & General Properties
+----------
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**Project ID**: Google Cloud Project ID, which uniquely identifies a project.
+It can be found on the Dashboard in the Google Cloud Platform Console.
+
+**GCS Bucket as Buffer:** The name of the GCS bucket serving as the buffer for the Anaplan data file export.
+
+**File archive name in GCS bucket:** The desired file name for the data export from Anaplan model.
+
+**Format:** Format of the data that export from the Anaplan model.
+The format must be one of 'avro', 'blob', 'csv', 'delimited', 'json', 'parquet', 'text', 'tsv', or the
+name of any format plugin that you have deployed to your environment.
+If the format is a macro, only the pre-packaged formats can be used.
+If the format is 'blob', every input file will be read into a separate record.
+The 'blob' format also requires a schema that contains a field named 'body' of type 'bytes'.
+If the format is 'text', the schema must contain a field named 'body' of type 'string'.
+
+**Delimiter:** Delimiter to use when the format is 'delimited'. This will be ignored for other formats.
+
+**Skip Header** Whether to skip the first line of each file. Supported formats are 'text', 'csv', 'tsv', 'delimited'.
+
+**Service Account**  - service account key used for authorization
+
+* **File Path**: Path on the local file system of the service account key used for
+authorization. Can be set to 'auto-detect' when running on a Dataproc cluster.
+When running on other clusters, the file must be present on every node in the cluster.
+
+* **JSON**: Contents of the service account JSON file.
```

Comment:
Is this for capturing the actual content of the json key file?

Reply:
Yeah, it is.